eng/Versioning.targets

```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
+  <PropertyGroup Condition="'$(SkipDevBuildNumber)' != 'true'">
+    <_BuildNumber>$(OfficialBuildId)</_BuildNumber>
+    <_BuildNumber Condition="'$(OfficialBuildId)' == ''">$([System.DateTime]::Now.ToString(yyyyMMdd)).1</_BuildNumber>
```

Comment:
Shouldn't the ".1" be the build revision number, though I'm not sure if that's available during build. In the legacy release pipelines, it was `$(Rev:r)` or `$(Rev:rr)` for double digits (good, if you expect that volume, since according to semver any prerelease metadata is lexigraphically sorted). That said, pipeline templates can define and use a [counter](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#counter) if `$(Rev)` is not available.

Reply:
That is what is coming into OfficialBuildId property but for local dev builds I just hard-code that to 1.